rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -302,6 +308,63 @@ pub struct Neuron {
     /// Cf. \[Neuron::stop_dissolving\] and \[Neuron::start_dissolving\].
     #[prost(oneof = "neuron::DissolveState", tags = "9, 10")]
     pub dissolve_state: Option<neuron::DissolveState>,
+    /// The amount of "sway" this neuron has when voting on proposals.
+    ///
+    /// When a proposal is created, each eligible neuron gets a "blank" ballot. The
+    /// amount of voting power in that ballot is set to the neuron's deciding
+    /// voting power at the time of proposal creation. There are two ways that a
+    /// proposal can become decided:
+    ///
+    ///   1. Early: Either more than half of the total voting power in the ballots
+    ///   votes in favor (then the proposal is approved), or at least half of the
+    ///   votal voting power in the ballots votes against (then, the proposal is
+    ///   rejected).
+    ///
+    ///   2. The proposal's voting deadline is reached. At that point, if there is
+    ///   more voting power in favor than against, and at least 3% of the total
+    ///   voting power voted in favor, then the proposal is approved. Otherwise, it
+    ///   is rejected.
+    ///
+    /// If a neuron regularly refreshes its voting power, this has the same value
+    /// as potential_voting_power. Actions that cause a refresh are as follows:
+    ///
+    ///     1. voting directly (not via following)
+    ///     2. set following
+    ///     3. refresh voting power
+    ///
+    /// (All of these actions are performed via the manage_neuron method.)
+    ///
+    /// However, if a neuron has not refreshed in a "long" time, this will be less
+    /// than potential voting power. See VotingPowerEconomics. As a further result
+    /// of less deciding voting power, not only does it have less influence on the
+    /// outcome of proposals, the neuron receives less voting rewards (when it
+    /// votes indirectly via following).
+    ///
+    /// For details, see https://dashboard.internetcomputer.org/proposal/132411.
+    ///
+    /// Per NNS policy, this is opt. Nevertheless, it will never be null.
+    #[prost(uint64, optional, tag = "26")]
+    pub deciding_voting_power: Option<u64>,
```

Comment:
I wonder if a better name for this field could be selected. 

In the motion proposal linked above, this seems to be called "adjusted" voting power. 

An even better name, IMO, would be 

```suggestion
    pub effective_voting_power: Option<u64>,
```

Reply:
> has been agreed upon with clients

Okay, then this comment is NCR.